.github/workflows/release-embedding-sdk.yml

```diff
@@ -37,8 +37,138 @@ jobs:
           triggered by: @${{ github.event.sender.login }}
           EOF
 
-  determine-version:
-    name: Determine SDK version
+  send-slack-notification-start:
+    name: "Send Slack notification: SDK release started"
+    needs: determine-sdk-version
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    outputs:
+      slackMessageId: ${{ steps.send-slack-message-start.outputs.result }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: release
+
+      - name: Prepare build scripts
```

Comment:
We'll only build the build scripts once and cache it as an artifact, this saves like 10-20s depending on how fast the build is.

I could technically re-build the build script every time, but why wasting time 🤷

Reply:
for 20 seconds I wonder if we should favor simplicity over speed?

Reply:
I think uploading built files is quite common when you need files to persist between jobs. It's actually simpler. To build, upload, then redownload as many times, especially in this case, we just need one single javascript file.

Duplications could be seen simpler in terms of consistency, but I think I'll follow [the CI practice](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow#about-workflow-artifacts), since the overhead is almost non-existence.